hpat/_str_decode.cpp

```diff
@@ -377,8 +370,6 @@ void decode_utf8(const char *s, Py_ssize_t size, int* kind, int *is_ascii, int*
                 goto End;
             }
             errmsg = "invalid continuation byte";
-            startinpos = s - starts;
-            endinpos = startinpos + ch - 1;
             break;
```

Comment:
How close is this code to the original CPython version?

Reply:
Not sure about CPython but these variables used nowhere.